README.md

```diff
@@ -49,12 +49,13 @@ Segments are pre-defined groups of customers that features can be targeted to. M
 
 ### Methods
 
-#### poll
+#### initialize
 
-Start polling for raw configuration data every `pollingRateSeconds`:
+Make an initial request for changes and start polling for raw configuration data
+every `pollingRateSeconds`:
 
 ```js
-poller.poll()
+poller.initialize()
```

Comment:
does that mean client can still call poller.poll()? (imagine I call `initialize()` first, and then stop it, the next time I should call `poll()` ?)

Reply:
they could call `poll` after `stop` since the cache will still be populated

Reply:
should we then not delete the poll() in readme?

Reply:
hm, i'd rather just keep it simple with `start` and `stop` there's no performance impact or anything from calling `start` again after `stop`